test_agent/python/testagent.py

```diff
@@ -65,31 +60,36 @@
 logger = logging.getLogger("File:Line# Debugger")
 logger.setLevel(logging.DEBUG)
 
+RESPONSE_URI = UUri(ue_id=1, ue_version_major=1, resource_id=0)
+
+sdkname = "python"
+transport_name = "socket"
+
 
 class SocketUListener(UListener):
     def on_receive(self, umsg: UMessage) -> None:
```

Comment:
use try and except to handle errors

Reply:
not sure where specifically you're talking about